inttest/basic/basic_test.go

```diff
@@ -212,14 +231,14 @@ func (s *BasicSuite) verifyContainerdDefaultConfig() {
 	}).URI(), parsedConfig.Plugins.CRI.SandboxImage)
 }
 
-func (s *BasicSuite) verifyCoreDNSAntiAffinity(kc *kubernetes.Clientset) {
+func (s *BasicSuite) checkCoreDNSAntiAffinity(ctx context.Context, kc *kubernetes.Clientset) error {
```

Comment:
let's either remove the verb "check" from the function name or get assertions s.NoError back? 

having in the name "verify" or "check" imo implies that something would be asserted inside

Reply:
I get your point, although I don't really have a super good idea for a better fitting verb. Maybe "probe"?